static/app/views/insights/database/queries/useOutdatedSDKProjects.tsx

```diff
@@ -35,10 +41,7 @@ export function useOutdatedSDKProjects(options?: Options) {
 
       return semverCompare(update.sdkVersion, minimumRequiredVersion) === -1;
     })
-    .map(update => update.projectId)
-    .map(projectId => {
-      return ProjectsStore.getById(projectId);
-    })
```

Comment:
collapsed this because lint was complaining that `projectId` exists in the upper scope. Also, we don't need to loop twice.